RELEASE/scripts/autoscend.ash

```diff
@@ -4765,11 +4772,10 @@ void print_header()
 boolean doTasks()
 {
 	if(get_property("_casualAscension").to_int() >= my_ascensions())
-	{
-		auto_log_warning("I think I'm in a casual ascension and should not run. To override: set _casualAscension = -1", "red");
-		return false;
+	{	
+		auto_log_warning("I think I'm in a casual ascension and should not run. To override: set _casualAscension = -1", "red");	
+		return false;	
```

Comment:
whitespace has been changed here. Be careful when copy & pasting stuff from the GitHub UI into source code as it will do this sort of thing.

Reply:
Right thank you. I will be more careful in the future about this.

Reply:
Should I go ahead and commit this branch to beta? or should try to undo the whitespace changes via more commits to this branch?

Reply:
It's approved, commit it. If I was that bothered about the whitespace I'd ask for changes but it's fine (thankfully ASH is not Python or other stupid languages which care about whitespace).

Reply:
Alright, thank you